.dockerignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I think we should also include other directories listed in the `.gitignore` that might contain a quite some files.

```suggestion
# Compiled output
/dist
/tmp
/out-tsc
/bazel-out
/docs

# Node
/node_modules
```